.github/workflows/pull_request_examples.yml

```diff
@@ -97,19 +99,24 @@ jobs:
                   cd '${{ github.workspace }}'
                   npm ci --workspaces --include-workspace-root
                   npm run --if-present build
+                  mkdir -p snapwp-test-app
 
-            - name: Install dependencies
-              working-directory: ${{ github.workspace }}/examples/${{ matrix.example }}
-              run: npm install
-
-            - name: Copy .env.example to .env
-              working-directory: ${{ github.workspace }}/examples/${{ matrix.example }}
+            - name: Scaffold Snapwp App
+              working-directory: ${{ github.workspace }}/snapwp-test-app
               run: |
-                  cp .env.example .env
+                  cp '${{ github.workspace }}/examples/${{ matrix.example }}/.env.example' .env
                   sed -i 's|NEXT_PUBLIC_WP_HOME_URL=.*|NEXT_PUBLIC_WP_HOME_URL=http://localhost|' .env
+                  npm i '${{ github.workspace }}/packages/cli' --no-save
+                  npx snapwp <<< '.'
+                  npm i
+                  for i in '${{ github.workspace }}/packages/'*; do
+                      if [ -d "$i" ]; then
+                          npm i "$i"  # Swap snapwp dependencies for local packages
+                      fi
+                  done
```

Comment:
@ayushnirwal / @SH4LIN if there's an _easy_ cleaner way to install local copies of the packages before I merge this spaghetti, please LMK.

Ref: https://rtcamp.slack.com/archives/CK2CEH4F6/p1744296854287099?thread_ts=1744273250.043439&cid=CK2CEH4F6